v2/pkg/requests/bulk-http-request.go

```diff
@@ -87,7 +87,7 @@ func (r *BulkHTTPRequest) MakeHTTPRequest(baseURL string, dynamicValues map[stri
 	if err != nil {
 		return nil, err
 	}
-	hostname := parsed.Hostname()
+	hostname := parsed.Host
```

Comment:
Just saw this and recalled #166: what if we introduce a new variable `Host` with value `parsed.Host` and then leave everything else untouched? Looks like it would nicely fit to reflect the composite value `hostname[:port]`.

Reply:
Thanks for the suggestion @manuelbua - We are going to merge this PR, could you open a new issue so that we can adequately discuss it?